debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlDecimalIT.java

```diff
@@ -49,6 +49,7 @@ public void beforeEach() {
         DATABASE.createAndInitialize();
         initializeConnectorTestFramework();
         Testing.Files.delete(DB_HISTORY_PATH);
+        skipAvroValidation();
```

Comment:
Can you add a comment here about the reasoning? I see it's mentioned below, but I feel calling such method should justified always at the call site itself as there may be different reasons.